int32 id
float64 time
float64 coordinate_x
float64 coordinate_y
float64 coordinate_z